Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -63,5 +63,15 @@ dotnet publish $VFS_SRCDIR/GVFS.sln /p:Configuration=$CONFIGURATION.Mac /p:Platf
 echo 'Copying Git installer to the output directory...'
 $VFS_SCRIPTDIR/PublishGit.sh $GITPATH || exit 1
 
+echo 'Installing shared data queue stall workaround...'
+# We'll generate a temporary project if and only if we don't find the correct dylib already in place.
+BUILDDIR=$VFS_OUTPUTDIR/GVFS.Build
+if [ ! -e $BUILDDIR/libSharedDataQueue.dylib ]; then
```

Comment:
Slick, this is a nice optimization! Do we want to do the same thing in the DownloadGVFSGit.sh script at this time?

Reply:
At this time, no, since it's not required for this change. I'll add it to my cleanup todo list.